security-force-monitor/sfm-cms
# Update About page and redirect Help, Countries to it

## Overview

Redirect `/help` and `/countries` to `/about`, and update the text on `/about`.

Connects https://github.com/security-force-monitor/sfm-cms/issues/675.

## Testing Instructions

* Visit the homepage at http:localhost:8000 and confirm that you no longer see `Countries` or `Help` in the navbar
* Visit http://localhost:8000/help and http://localhost:8000/countries and confirm you get redirected to http://localhost:8000/about
* Read the text on http://localhost:8000/about and confirm it looks good and links resolve properly
